### DISCUSSION TITLE
URL to docs for backup is wrong?

### DISCUSSION THREAD

**Discussion by gilgongo** [pivpn/pivpn#1777] [Bug Reports]
Hi - I notice the URL given after the `-bk` option seems wrong. At least, there's nothing on that page about restoring backups.

```
:~$ pivpn -bk
Backup created in /home/gilgongo/pivpnbackup/20231104-162644-pivpnwgbackup.tgz 
To restore the backup, follow instructions at:
https://docs.pivpn.io/openvpn/wireguard/#migrating-pivpn-wireguard
```

Should the URL be this one?

https://docs.pivpn.io/wireguard/#migrating-pivpn-wireguard

**Comment by orazioedoardo** [comment]
Thanks, done: https://github.com/pivpn/pivpn/commit/0a3dd62ee703bb55ef1fc1fd0bfd1c29f6acc36b
